server/config/routes.js

```diff
@@ -951,6 +951,7 @@ exports.configure = function configure(app) {
   app.put('/lots/:uuid', lots.update);
   app.get('/lots/:uuid/assignments/:depot_uuid', lots.assignments);
   app.post('/lots/:uuid/merge', lots.merge);
+  app.post('/lots/merge/auto', lots.autoMerge);
   app.get('/inventory/:uuid/lot_candidates', lots.getCandidates);
   app.get('/lots_dupes/:label?/:entry_date?/:expiration_date?/:initial_quantity?/:inventory_uuid?', lots.getDupes);
```

Comment:
Let's discuss this on Skype sometime.  I actually think you can remove everything here and just leave:
```js
   app.get('/lots_dupes/', lots.getDupes); 
```

but I'm curious where the confusion is coming in.  Note that this may be outside the scope if this PR if it's not as straightforwards as I just mentioned.